src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetNodeComputer.java

```diff
@@ -40,8 +40,15 @@ public EC2FleetNode getNode() {
     @Nonnull
     @Override
     public String getDisplayName() {
+        if(cloud != null) {
+            final String displayName = String.format("%s %s", cloud.getDisplayName(), name);
+            if (getNode() != null && getNode().getMaxTotalUses() != -1) {
+                return String.format("%s Builds left: %d ", displayName, getNode().getMaxTotalUses());
```

Comment:
does using newline `\n` work for separating the builds from the display name? curious if this ever renders depending on screen size etc.

Reply:
nah, it does not add a newline. The screenshot just happens to push it to next line. I tried `%n` `/n` `System.getProperty("line.separator")` but somehow none of these work. I assumed having it space separated would just be fine.

Reply:
certainly better than nothing